src/tests/ftest/fault_domain/fault_domain.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from apricot import TestWithServers, skipForTicket
+
+
+class FaultDomain(TestWithServers):
+    """Test class Fault domains
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up fault domain test."""
+        self.start_agents_once = False
+        self.start_servers_once = False
+        self.setup_start_agents = False
+        self.setup_start_servers = False
+        super().setUp()
+
+    @skipForTicket("DAOS-7919")
+    def test_pools_in_different_domains(self):
+        """The aims to:
+            Be able to configure daos servers using different fault domains.
+            Created pools must be in different fault domains.
+
+           Not coded, should be available when DAOS-7919 is resolved.
+           If a pool has the same fault domains, a debug message
+             should be expected.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large
+        :avocado: tags=fault_domain,fault_domain_different_domains
+        """
+        test_passed = True
+        error_messages = list()
+        fault_paths = self.params.get("fault_path", '/run/*')
+
+        for counter, server in enumerate(self.hostlist_servers):
+            self.add_server_manager()
+            self.configure_manager(
+                "daos_server",
+                self.server_managers[counter],
+                [server],
+                self.hostfile_servers_slots
+            )
+            self.server_managers[counter].set_config_value("fault_path",
+                                                           fault_paths[counter])
+
+        # Servers are started
+        for server in self.server_managers:
+            server.start()
+
+        self.start_agents()
+
+        # Create pools, setting the values obtained from yaml file
+        self.pool = []
+        for index in range(5):
```

Comment:
Minor: If you intend to create 5 pools, it will be nice to add this to yaml file so that we can control the pool_count from yaml file if ever needs to be changed.